test.js

```diff
@@ -73,6 +73,13 @@ test.serial('don\'t kill self', async t => {
 	Object.defineProperty(process, 'pid', {value: originalFkillPid});
 });
 
+test.serial('don\'t kill `fkill` when killing `node`', async t => {
+	const originalFkillPid = process.pid;
+	await fkill('node');
```

Comment:
Sorry if I may have missed the point of this PR but I don't understand how the this test is meant to work on Windows.

On Windows, if we try to kill `node` it won't find any process because the process name for the Node.js runtime it's `node.exe`, hence the test would pass because it won't kill `fkill` yes, but it also won't kill any other `node` processes as well, which kind of make it pointless to try killing by process name in the first place.

As an example, try running on a console `node -e "while(true){}"` and try running the test, which should kill all `node` processes, but it won't on Windows.

Reply:
^ @yaodingyd

Reply:
https://github.com/sindresorhus/fkill/pull/38